### ISSUE TITLE
Error when running code in “QuickStart”

### ISSUE THREAD

**Issue by EricPengShuai** [matthias-k/pysaliency#17]
Hello! @matthias-k 

After installing pysaliency package，run code in your “QuickStart”，the following error occurred.
```python
In [4]: import pysaliency

In [5]: dataset_location = 'datasets'
   ...: model_location = 'models'
   ...:
   ...: mit_stimuli, mit_fixations = pysaliency.external_datasets.get_mit1003(location=dataset_location)
   ...: aim = pysaliency.AIM(location=model_location)
   ...: saliency_map = aim.saliency_map(mit_stimuli.stimuli[0])
   ...:
   ...: plt.imshow(saliency_map)
   ...:
   ...:
   ...: auc = aim.AUC(mit_stimuli, mit_fixations)
Downloading file:  16%|███████▋                                         | 36.8M/235M [01:59<10:41, 309kB/s]
Checking md5 sum...
~/Anaconda/anaconda3/envs/pytorch_dl/lib/python3.6/site-packages/pysaliency/utils.py:327: UserWarning: MD5 sum of /var/folders/8n/d46946h949n5jj4ndz0q4khr0000gn/T/tmp7rcls1ne/ALLSTIMULI.zip has changed. Expected 0d7df8b954ecba69b6796e77b9afe4b6 but got 78bc4e1d01996155ed7dc0267b751429. This might lead to this code producing wrong data.
  " this code producing wrong data.".format(filename, md5_hash, file_hash))
Downloading file: 47.1MB [00:44, 1.05MB/s]
Checking md5 sum...
Downloading file: 32.8kB [00:00, 318kB/s]
Checking md5 sum...
Creating stimuli
---------------------------------------------------------------------------
BadZipFile                                Traceback (most recent call last)
<ipython-input-5-e4d86b1dc162> in <module>
      2 model_location = 'models'
      3
----> 4 mit_stimuli, mit_fixations = pysaliency.external_datasets.get_mit1003(location=dataset_location)
      5 aim = pysaliency.AIM(location=model_location)
      6 saliency_map = aim.saliency_map(mit_stimuli.stimuli[0])
```

**Comment by matthias-k** [comment]
That looks like the download did not complete or was otherwise corrupted. I just checked and I can download the files without getting an MD5 checksum warning. I'll close the issue for now feel free to reopen it if the issue turns out to be something else.

**Comment by EricPengShuai** [comment]
hello, @matthias-k 

execute me, after I run the code again, the download procedure complete successfully, but `FileNotFoundError` occurred.
```python
FileNotFoundError: [Errno 2] No such file or directory: '~/Anaconda/anaconda3/envs/pytorch_dl/lib/python3.6/site-packages/pysaliency/scripts/extract_fixations.m'
```

I am wondering if it was a problem with the pip installation package. I use the following command:
```python
pip install Cython -i http://pypi.douban.com/simple --trusted-host pypi.douban.com
pip install pysaliency -i http://pypi.douban.com/simple --trusted-host pypi.douban.com
```

Thanks,
EricPengShuai

**Comment by matthias-k** [comment]
Hi @EricPengShuai,

thanks for the report. It turns out that since I switched to setuptools, the matlab scripts where missing in the python package. I fixed this and just uploaded version 0.2.21 to pypi. With this update, it should work.

**Comment by EricPengShuai** [comment]
@matthias-k 
Do I need to install matlab or octave to use pysaliency? After I run the QuickStart code again, it turned out that：
```python
Downloading file: 235MB [00:21, 10.8MB/s]                                                                                                                      
Checking md5 sum...
Downloading file: 47.1MB [00:10, 4.46MB/s]                                                                                                                     
Checking md5 sum...
Downloading file: 32.8kB [00:00, 218kB/s]                                                                                                                      
Checking md5 sum...
Creating stimuli
Creating fixations
Running original code to extract fixations. This can take some minutes.
Warning: In the IPython Notebook, the output is shown on the console instead of the notebook.
Traceback (most recent call last):
  File "/Desktop/dataset/start.py", line 7, in <module>
    mit_stimuli, mit_fixations = pysaliency.external_datasets.get_mit1003(location=dataset_location)
  File "~/Anaconda/anaconda3/envs/pytorch_dl/lib/python3.6/site-packages/pysaliency/external_datasets.py", line 505, in get_mit1003
    return _get_mit1003('MIT1003', location=location, include_initial_fixation=False)
  File "~/Anaconda/anaconda3/envs/pytorch_dl/lib/python3.6/site-packages/pysaliency/external_datasets.py", line 360, in _get_mit1003
    run_matlab_cmd('extract_all_fixations;', cwd=temp_dir)
  File "~/Anaconda/anaconda3/envs/pytorch_dl/lib/python3.6/site-packages/pysaliency/utils.py", line 302, in run_matlab_cmd
    matlab = get_matlab_or_octave()
  File "~/Anaconda/anaconda3/envs/pytorch_dl/lib/python3.6/site-packages/pysaliency/utils.py", line 298, in get_matlab_or_octave
    raise Exception('No version of matlab or octave was found on this system!')
Exception: No version of matlab or octave was found on this system!
```
